packages/main/test/pages/Popover.html

```diff
@@ -531,6 +531,7 @@ <h2>Open and scroll</h2>
 	<br><br>
 	<br>
 
+
```

Comment:
No empty line was here before, I suppose it is leftover from testing. Remove this and below, otherwise +2